src/components/ShoppingCart/CartListItem.tsx

```diff
@@ -4,6 +4,7 @@ import { FaPen } from "react-icons/fa";
 import { MdDeleteForever } from "react-icons/md";
 import { ICartItem } from "@/interfaces/cartItem";
 import DropDownButton from "@/components/ShoppingCart/dropDownButton";
+import { WarningIcon } from "@chakra-ui/icons";
```

Comment:
It is not a problem caused by y'all but we do have another icon library installed already in this repo called "react-icons". It has more icons and is also easy to use. Please remove "chakra-ui/icons" from package.json and use an icon from "react-icons" here.

All icons and document of this lib can be found here.
https://react-icons.github.io/react-icons/

Reply:
Thanks for your advice. I've updated the icon and removed "chakra-ui/icons" from the package.